third_party/picosoc_wrappers/README.chromium

```diff
@@ -0,0 +1,11 @@
+Name: PicoSoC - A simple example SoC using PicoRV32
```

Comment:
You don't want to call it `README.chromium` should be named `README.fpga-tool-perf`

Reply:
Sure, done